src/template.css

```diff
@@ -32,6 +32,280 @@ a:hover {
+    color: black;
+}
+
+
+#keyboardControlsWindow .row .form-inline .form-group .form-control{
+    font-size: 21px;
+}
+
+.h6, h6 {
+    font-size: 150% !important;
+    background-color: black;
+    background-color: #00000057;
+}
+
+.btn{
+  font-size: 25px;
+}
+#romFile{
+  font-size: 20px;
+}
+
+
+/* Buttons */
+
+.creditsbutton:hover{
+    color: rgba(255, 255, 255, .5) !important;
+    cursor: pointer !important;
+}
+
+#googleSigninButton:hover{
+    cursor: pointer;
+}
+
+#discordSigninButton:hover{
+    cursor: pointer;
+}
+
+#startbutton{
+  background-image: url("/mmo/assets/btn-stone-edit.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  font-weight: 348;
+  color: white;
+}
+
+.btn-primary {
+    color: #fff;
+    background-image: url("/mmo/assets/btn-blue-edit.png");
+    background-size: 100% 100%;
+    border: none;
+}
+
+.btn-primary:disabled{
+  color: #00000094;
+  background-image: url("/mmo/assets/btn-stone-edit.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  text-shadow: none;
+}
+
+.btn-success {
+    color: #fff;
+    background-image: url("/mmo/assets/btn-green-edit.png");
+    background-size: 100% 100%;
+    border: none;
+}
+
+.btn-success:disabled{
+  color: #00000094;
+  background-image: url("/mmo/assets/btn-stone-edit.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  text-shadow: none;
+}
+
+.btn-warning {
+    color: #fff;
+    background-image: url("/mmo/assets/btn-pink-edit.png");
+    background-size: 100% 100%;
+    border: none;
+}
+
+.btn-warning:disabled{
+  color: #00000094;
+  background-image: url("/mmo/assets/btn-stone-edit.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  text-shadow: none;
+}
+
+.btn-danger {
+    color: #fff;
+    background-image: url("/mmo/assets/btn-red-edit.png");
+    background-size: 100% 100%;
+    border: none;
+}
+
+.btn-danger:disabled{
+  color: #00000094;
+  background-image: url("/mmo/assets/btn-stone-edit.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  text-shadow: none;
+}
+
+.btn{
+  border-radius: 1px;
+  line-height: 132%;
+  text-shadow: #00000052 2px 2px 1px;
+}
+
+#logoutButton{
+  background-image: url("/mmo/assets/btn-stone-edit.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  font-weight: 348;
+}
+
+.btn-secondary{
+  background-image: url("/mmo/assets/btn-stone-edit.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  font-weight: 348;
+}
+
+/* More stuff */
+
+.signboxBackground{
+  padding: 14px;
+  background-image: url("/mmo/assets/yellow-background-wide.png");
+  background-size: 100% 100%;
+  border: none;
+  opacity: 1;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.form-control-sm {
+    font-size: 20px;
+}
+
+#discordNameBox{
+    font-size: 19px;
+}
+
+/* game canvas and chat changes, probably not the best way to do this */
+.chatboxPos{
+margin-left: 564px;
```

Comment:
you should definitely not use such huge margins. Instead either use flexbox directly or since we use bootstrap, you should be using their classes to also support smaller screen sizes.

Reply:
same comment as in line 227